core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -869,7 +869,7 @@ public static String getOncoKBPublicApiUrl()
         
         // Empty string should be used if you want to disable the OncoKB annotation.
         if(oncokbApiUrl == null || oncokbApiUrl.isEmpty()) {
-            oncokbApiUrl = "oncokb.org/api/v1";
+            oncokbApiUrl = "www.oncokb.org/api/v1";
```

Comment:
The comment above the if statement is incorrect ... you can never disable OncoKB annotation because of this code. I would add a symbolic static final string at the top of the class (DEFAULT_ONCO_KB_API_URL)